jovian/tests/utils/test_slug.py

```diff
@@ -0,0 +1,14 @@
+from jovian.utils.slug import get_current_slug, set_current_slug
+from jovian.utils import slug
+from unittest import mock
+
+
+@mock.patch("jovian.utils.slug._current_slug", "test_slug")
+def test_get_current_slug():
+    assert get_current_slug() == "test_slug"
+
+
+@mock.patch("jovian.utils.slug.get_notebook_slug", return_value="test_slug")
+def test_set_current_slug(mock_get_notebook_slug):
+    set_current_slug("fake_filename")
+    assert slug._current_slug == "test_slug"
```

Comment:
This test can be improved. We want to test that `set_current_slug` (ideally it should be called `detect_current_slug`) gets the slug for the given `filename` from the `.jovianrc` file. However, by mocking `get_notebook_slug`, we are completely ignoring the `filename` argument. 

Try mocking the `rcfile` instead of mocking `get_notebook_slug`, or mock `get_notebook_slug` so that it returns `test_slug` only if the argument is `fake_filename`